app/src/main/res/values-pt-rBR/strings.xml

```diff
@@ -188,56 +201,56 @@
     <item quantity="other">\n      %s Lições\n    </item>
   </plurals>
   <plurals name="completed_story_count">
-    <item quantity="one">1 História Concluída\n    </item>
-    <item quantity="zero">%s Histórias Concluídas\n    </item>
-    <item quantity="other">\n      %s Histórias Concluídas\n    </item>
+    <item quantity="one">1 história concluída\n    </item>
+    <item quantity="zero">%s Histórias concluídas\n    </item>
+    <item quantity="other">\n      %s Histórias concluídas\n    </item>
   </plurals>
   <plurals name="ongoing_topic_count">
-    <item quantity="one">1 Tópico em Andamento\n    </item>
-    <item quantity="zero">%s Tópicos em Andamento\n    </item>
-    <item quantity="other">\n      %s Tópicos em Andamento\n    </item>
+    <item quantity="one">1 Tópico em andamento\n    </item>
+    <item quantity="zero">%s Tópicos em andamento\n    </item>
+    <item quantity="other">\n      %s Tópicos em andamento\n    </item>
```

Comment:
Invalid new lines

Reply:
Similarly this is what the English text looks like:

![Screenshot from 2023-09-29 19-54-32](https://github.com/oppia/oppia-android/assets/10575562/8922280f-e357-4283-8654-e988a7516fde)